src/features/main/lib/validators.ts

```diff
@@ -0,0 +1,33 @@
+import { z } from 'zod';
+
+export const RoomModeEnum = z.enum(['SOLO', 'TEAM']);
+
+export const createRoomSchema = z
+  .object({
+    name: z
+      .string()
+      .trim()
+      .min(1, '방 제목을 입력해주세요.')
+      .max(15, '최대 15자까지 입력 가능합니다.'),
+
+    mode: RoomModeEnum,
+
+    maxPlayers: z.number(),
+
+    isPrivate: z.boolean(),
+
+    password: z.string().optional(),
+  })
+  .superRefine((data, ctx) => {
```

Comment:
```suggestion
  .object({
    name: z
      .string()
      .trim()
      .min(1, '방 제목을 입력해주세요.')
      .max(15, '최대 15자까지 입력 가능합니다.'),
    mode: RoomModeEnum,
    maxPlayers: z.number(),
    isPrivate: z.boolean(),
    password: z.string().optional(),
  })
  .superRefine((data, ctx) => {
```
띄어쓰기 공백은 제거해 주시는게 가독성에 도움이 될 것 같습니다😁